libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -18,19 +18,27 @@ const AP_Param::GroupInfo AP_Mission::var_info[] = {
     AP_GROUPINFO_FLAGS("TOTAL",  0, AP_Mission, _cmd_total, 0, AP_PARAM_FLAG_INTERNAL_USE_ONLY),
 
     // @Param: RESTART
-    // @DisplayName: Mission Restart when entering Auto mode
+    // @DisplayName: Mission - Restart Behaviour when entering AUTO mode
     // @Description: Controls mission starting point when entering Auto mode (either restart from beginning of mission or resume from last command run)
-    // @Values: 0:Resume Mission, 1:Restart Mission
+    // @Range: 0 3
+    // @Values: 0:Resume Mission at the WayPoint it has been suspended, 1:Restart Mission, 2:Restart PARALLEL translated Relative Mission, 3:Restart ROTATED  translated Relative Mission
     // @User: Advanced
     AP_GROUPINFO("RESTART",  1, AP_Mission, _restart, AP_MISSION_RESTART_DEFAULT),
 
     // @Param: OPTIONS
-    // @DisplayName: Mission options bitmask
+    // @DisplayName: Mission - options bitmask
     // @Description: Bitmask of what options to use in missions.
-    // @Bitmask: 0:Clear Mission on reboot, 1:Use distance to land calc on battery failsafe
+    // @Bitmask: 0:Clear Mission on reboot, 1:Use distance to land calc on battery failsafe, Bit 2: At restart skip altitude at first Waypoint, Bit 3: At restart use altitude offset (Start-Waypoint to first Waypoint) for all Waypoints
```

Comment:
The parameter parse test is failing probably because of the use of word "Bit" here.  Only the integer is required, not the word bit itself.

Reply:
Thank you @rishabsingh3003 , I will fix that.
How did you find that? How can I deepdive into the test-results to find that erroneous line?

Reply:
You can go through this link here:
https://ardupilot.org/dev/docs/the-ardupilot-autotest-framework.html
I am new to ArduPilot myself, but what I usually do to find the failed test, is to first see the name of the test that is being reported. You can do that by scrolling all the way down, at whatever build that's failing. Then go through the [auto-test code](https://github.com/ArduPilot/ardupilot/tree/master/Tools/autotest) to see what exactly it is testing, to figure out what has failed for my code in review. 

Also, another small advice would be to squash all your commits into one, if you fix something in your PR, instead of adding another commit, and then force push it onto your branch. I believe this helps in not polluting the commit history. 
This link might be of some use to you:https://ardupilot.org/dev/docs/submitting-patches-back-to-master.html
I see all the tests are now passing so we've already made progress!
Hope this helps!

Reply:
@rishabsingh3003 Thank you very much, your hint solved the problem :o)
Additionally I will go through your links to gain a little experience for the next issue with the checks.